src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -816,6 +816,22 @@ public static void EndsWith(string s, string value, StringComparison comparisonT
         }
 
         [Theory]
+        [InlineData("Hello", 'o', true)]
+        [InlineData("Hello", 'O', false)]
+        [InlineData("o", 'o', true)]
+        [InlineData("o", 'O', true)]
```

Comment:
Why is this true? I believe we want to match .EndsWith(string) behavior which is case senseitive

https://msdn.microsoft.com/en-us/library/2333wewz(v=vs.110).aspx